### DISCUSSION TITLE
colour change while using cdn

### DISCUSSION THREAD

**Discussion by ilyasbabu** [simple-icons/simple-icons#6895] [Help with SVGs] [answered]
https://cdn.jsdelivr.net/npm/simple-icons@v3/icons/instagram.svg

by default this is white while using cdn in html pages. how can i turn this to white?

**Comment by sachinraja** [comment]
I don't believe that's possible. You can copy the svg and change the `fill` if you want a different color.

**Comment by Andre601** [comment]
Through random luck did I find https://simpleicons.vercel.app, which seems to offer what you are looking for.

In your case could you use https://simpleicons.vercel.app/instagram/fff for this output:
![insta](https://simpleicons.vercel.app/instagram/fff)

**Reply by sachinraja** [reply]
Awesome! Maybe we should consider hosting something like this?

**Reply by ilyasbabu** [reply]
Thanks a lot! Exactly what meets my need.

**Reply by Andre601** [reply]
> Awesome! Maybe we should consider hosting something like this?

It sure would be useful. Especially when it would also include something like an optional size, as that would allow easy use inside markdown without the need to upload the files somewhere.

**Reply by sachinraja** [reply]
Great, I'll look into it

**Comment by sachinraja** [comment, marked as answer]
I've made a Cloudflare worker to change the attributes of an svg from a url. This way you can use whatever version of the icon you want and it's not limited to only Simple Icons. [Repo link](https://github.com/sachinraja/svg-rewriter)

You can also use the [Iconify](Iconify.design) API. Example request: https://api.iconify.design/simple-icons/cloudflare.svg?color=orange&height=100. I don't believe there's a way to specify a version though and the icon may be removed across breaking changes without warning.

cc @Andre601

**Reply by Andre601** [reply]
Interesting.
I recently made a version of that simplicons page I mentioned before which allows colored and resized Octicons.

I.e. https://octicons.andre601.ch/mark-github/fff/100 becomes

![octicon](https://octicons.andre601.ch/mark-github/fff/100)

**Reply by sachinraja** [reply]
I'm going to select this as the answer as https://github.com/simpleicons/simpleicons.now.sh is not always up to date and both svg rewriter and iconify work well here.

**Reply by LitoMore** [reply]
I created a new CDN service https://cdn.simpleicons.org due to the `simpleicons.now.sh` lack of maintenance.

See https://github.com/simple-icons/simple-icons#cdn-with-colors

**Reply by LitoMore** [reply]
Here is another solution by using CSS Mask:

See https://github.com/simple-icons/simple-icons/issues/8573#issuecomment-1523705900
